packages/drivers/odsp-driver-definitions/src/tokenFetch.ts

```diff
@@ -11,7 +11,10 @@ export interface TokenResponse {
 	/** Token value */
 	token: string;
 
-	/** Flag indicating whether token was obtained from local cache */
+	/**
+	 * Whether or not the token was obtained from local cache.
+	 * @remarks `undefined` indicates that it could not be determined whether or not the token was obtained this way.
```

Comment:
Reviewers please correct me if I am wrong, but this was the contract for the same concept on the azure side. I think it's important we call this out explicitly, since the framework consumes this.

Reply:
Looks good to me. The final goal was to make both the clients as similar as possible. I think it is okay to follow the similar contract as azure client (https://github.com/microsoft/FluidFramework/blob/main/packages/drivers/routerlicious-driver/src/tokens.ts#L32)